### ISSUE TITLE
server.reboot() minor enhancements

### ISSUE THREAD

**Issue by mkinney** [Fizzadar/pyinfra#264]
server.reboot() is looking good. I was experimenting using it for adhoc stuffs.

Can you:
1) Maybe rename the timeout variable as it looks like there is a connection timeout and wait for reboot timeout? (perhaps `reboot_timeout`)
2) Add the return code in debug output?
3) If `reboot_timeout=0`, make the operation be successful?

If you forget sudo, you cannot tell the command failed:

```
$ pyinfra -vv --debug 192.168.2.232 --user vagrant --password vagrant server.reboot delay=0 timeout=0
--> Loading config...
--> Loading inventory...
    [pyinfra_cli.inventory] Creating fake inventory...

--> Connecting to hosts...
    [pyinfra.api.connectors.ssh] Connecting to: 192.168.2.232 ({'allow_agent': False, 'look_for_keys': False, 'username': 'vagrant', 'timeout': 10, 'password': 'vagrant'})
    [192.168.2.232] Connected
    [pyinfra.api.state] Activating host: 192.168.2.232

--> Preparing operation...
    [pyinfra.api.operation] Adding operation, {'Server/Reboot'}, called @ /Users/mikekinney/Documents/python/mkinney_pyinfra/venv/lib/python3.7/site-packages/pyinfra-0.13.1-py3.7.egg/pyinfra_cli/main.py:507, opLines=(0, 507), opHash=4f1a31d38cc6d1864c30baed55b1ceaa44e30443
    The server.reboot operation is in beta!

--> Proposed changes:
    Ungrouped:
    [192.168.2.232]   Operations: 1   Commands: 2

--> Beginning operation run...
--> Starting operation: Server/Reboot ('delay=0',)
    [pyinfra.api.operations] Starting operation Server/Reboot on 192.168.2.232
    [pyinfra.api.util] Building command (shell_executable: sh): reboot
    [pyinfra.api.connectors.ssh] Running command on 192.168.2.232: (pty=None) sh -c reboot
[192.168.2.232] >>> sh -c reboot
    [192.168.2.232] Command timed out after 0s
    [192.168.2.232] Error
    [pyinfra.api.state] Failing hosts: 192.168.2.232
--> pyinfra error: No hosts remaining!
```

If you just want to reboot a remote instance, it shows as an error:
```
$ pyinfra -vv --debug --sudo 192.168.2.232 --user vagrant --password vagrant server.reboot delay=0 timeout=0
--> Loading config...
--> Loading inventory...
    [pyinfra_cli.inventory] Creating fake inventory...

--> Connecting to hosts...
    [pyinfra.api.connectors.ssh] Connecting to: 192.168.2.232 ({'allow_agent': False, 'look_for_keys': False, 'username': 'vagrant', 'timeout': 10, 'password': 'vagrant'})
    [192.168.2.232] Connected
    [pyinfra.api.state] Activating host: 192.168.2.232

--> Preparing operation...
    [pyinfra.api.operation] Adding operation, {'Server/Reboot'}, called @ /Users/mikekinney/Documents/python/mkinney_pyinfra/venv/lib/python3.7/site-packages/pyinfra-0.13.1-py3.7.egg/pyinfra_cli/main.py:507, opLines=(0, 507), opHash=4f1a31d38cc6d1864c30baed55b1ceaa44e30443
    The server.reboot operation is in beta!

--> Proposed changes:
    Ungrouped:
    [192.168.2.232]   Operations: 1   Commands: 2

--> Beginning operation run...
--> Starting operation: Server/Reboot ('delay=0',)
    [pyinfra.api.operations] Starting operation Server/Reboot on 192.168.2.232
    [pyinfra.api.util] Building command (shell_executable: sh sudo: True): reboot
    [pyinfra.api.connectors.ssh] Running command on 192.168.2.232: (pty=None) sudo -S -H -n sh -c reboot
[192.168.2.232] >>> sudo -S -H -n sh -c reboot
    [192.168.2.232] Command timed out after 0s
    [192.168.2.232] Error
    [pyinfra.api.state] Failing hosts: 192.168.2.232
--> pyinfra error: No hosts remaining!
```

**Comment by Fizzadar** [comment]
Have fixed the arg clash in https://github.com/Fizzadar/pyinfra/commit/f2c437649abd5c6f0b412bbf338bbc243e8fef45 & released in `v0.13.2`!

**Comment by mkinney** [comment]
Just tested. Works great.
